src/cli.rs

```diff
@@ -41,6 +42,25 @@ pub enum Cli {
         )]
         from: Option<String>,
 
+        /// Output a JSON summary to stdout containing the changes that happened
+        ///
+        /// This is made to be machine readable and piped into another tool.
+        /// For example you could send yourself notifications based on the output.
+        ///
+        /// Tip: When this option is used the status code will no longer be 1 when some site failed.
```

Comment:
Current behavior: exit code 1 when some site failed. As the exit code is not 0 in that case CI and other tools notice that something isn't right.

With `-o pipefail` it's more annoying and the `--json-summary` contains information about the failed ones.

https://github.com/EdJoPaTo/website-stalker/blob/9cdca0efa1206fec559fd3e8b2cccaca6a41e68f/src/main.rs#L263-L265

Reply:
I agree that it should not exit 1 one, when `--json-summary` is used. My question is purely on the documentation.
At least remove the `tip:` because that's not a tip but an important behavior change!